apps/frontend/src/components/Auth/apiClient.tsx

```diff
@@ -86,17 +93,55 @@ export class ApiClient {
     return this.get("/auth/timesheet") as Promise<string>;
   }
 
+
+  // a function to create user object from a passed in userId
+  public async createUser(userId: String): Promise<UserSchema> {
+    var userConverted = {}
+
+    await this.get(`/user/usersById?userIds[]=${userId}`).then((userList) => {
+
+      var userType = {}
+      
+      // set current user's type
+      if (userList[0].Type === 'breaktime-associate') {
+        userType = UserTypes.Associate
+      }
+      else {
+        userType = UserTypes.Supervisor
+      }
+
+      // create current user
+      userConverted = {
+        UserID: userList[0].userID,
+        FirstName: userList[0].firstName,
+        LastName: userList[0].lastName,
+        Type: userType
+      };
+
+    })
+    return userConverted
+  }
+
+  // a function that returns list of multiple users based on list of userIds passed in
+  public async getUsers(userIds: Array<String>): Promise<UserSchema[]> {
+    var allUsers = []
+
+    userIds.forEach(async (element) => {
```

Comment:
Nice! Good using the existing `getUser`/`createUser` function.

I think it could be a tiny bit cleaner with `userIds.map()` (still creates a new array, but keeps you from having to manually push to a new array), but just a small nit pick.